docs/rst/installation/linux_installation.rst

```diff
@@ -2,3 +2,292 @@
+
+        .. important::
+
+            The *Library* package is required for building the remain tool packages.
+
+    .. tab:: Building with CMake
+
+        *eProsima Fast DDS QoS Profiles Manager* tool suite is build using `CMake <https://cmake.org/>`_ following these steps:
+
+        **Library**
+
+        .. code-block:: bash
+
+            mkdir -p ~/fastdds_qosprofman_ws/build/lib
+            cd ~/fastdds_qosprofman_ws/build/lib
+            cmake ../../src/Fast-DDS-QoS-Profiles-Manager/lib -DCMAKE_INSTALL_PREFIX=../../install/lib
+            cmake --build . --target install
+
+        .. note::
+
+            If *Library* test suite compilation is needed, the enable test build flag `EPROSIMA_BUILD_TESTS` should be included while building the *Library* package:
+
+            .. code-block:: bash
+
+                cmake ../../src/Fast-DDS-QoS-Profiles-Manager/lib -DCMAKE_INSTALL_PREFIX=../../install/lib -DEPROSIMA_BUILD_TESTS=ON
+
+        **CLI**
+
+        .. code-block:: bash
+
+            mkdir -p ~/fastdds_qosprofman_ws/build/cli
+            cd ~/fastdds_qosprofman_ws/build/cli
+            export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:~/fastdds_qosprofman_ws/install/lib
+            cmake ../../src/Fast-DDS-QoS-Profiles-Manager/cli -DCMAKE_INSTALL_PREFIX=../../install/cli
+            cmake --build . --target install
+
+        .. note::
+
+            Note that the already installed *Library* path has been sourced in the *CLI* build process with the command:
+
+            .. code-block:: bash
+
+                export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:~/fastdds_qosprofman_ws/install/lib
+
+        **Documentation**
+
+        .. important::
+
+            If the *Library* path has been sourced in the previous step, there is no need to include it again in the environment variable ``CMAKE_PREFIX_PATH``.
+
+            If, on the other hand, it has not been sourced previously, as long as it is required also to build the *Documentation* package, it needs to be sourced.
+
+        .. code-block:: bash
+
+            mkdir -p ~/fastdds_qosprofman_ws/build/docs
+            cd ~/fastdds_qosprofman_ws/build/docs
+            cmake ../../src/Fast-DDS-QoS-Profiles-Manager/docs -DCMAKE_INSTALL_PREFIX=../../install/docs
+            cmake --build . --target install
+
+Run CLI example
+---------------
+
+The ``fastddsqosprof`` executable file is generated in the installation path.
+In order to run the CLI, please, ensure to source the environment:
+
+.. tabs::
+
+    .. tab:: Built with colcon
+
+        .. code-block:: bash
+
+            source ~/fastdds_qosprofman_ws/install/setup.bash
+
+    .. tab:: Built with CMake
+
+        .. code-block:: bash
+
+            export LD_LIBRARY_PATH=~/fastdds_qosprofman_ws/install/lib/lib
+            export PATH=$PATH:~/fastdds_qosprofman_ws/install/cli/bin
+
+*CLI* should display its software version by running:
+
+.. code-block:: bash
+
+    fastddsqosprof -v
+
```

Comment:
Maybe a note here redirecting to the CLI example folder (even if there is only one right now) could be useful.